src/include/duckdb/common/multi_file_reader_options.hpp

```diff
@@ -24,6 +24,8 @@ struct MultiFileReaderOptions {
 	bool auto_detect_hive_partitioning = true;
 	bool union_by_name = false;
 	bool hive_types_autocast = true;
+	bool hive_file_filter = false;
+	string input_file_pattern;
```

Comment:
maybe explain the purpose of this variable in a brief comment? It may not be immediately clear why we need to pass on the pattern here.